src/mono/msbuild/android/build/AndroidBuild.targets

```diff
@@ -221,20 +216,23 @@
 
     <MonoAOTCompiler
         AotModulesTablePath="$(_AotModuleTablePath)"
-        AsPrefix="$(_AsPrefixPath)"
+        AsName="$(_AsName)"
+        AsOptions="-target $(_Triple) -c -x assembler"
```

Comment:
If we want to enable `AsOptions` to be configurable by the user then we should not hardcode it here.
We could either set it in the same fashion as we are setting the `_LdOptions` so it is overwritable in the build, or 
we could completely remove it, since we are already passing `AsName` and `_Triple` to the task which is enough information to adjust the cmd line options passed to clang in the MonoAOTCompiler build task.